tfjs-backend-webgl/src/webgl_util.ts

```diff
@@ -368,8 +368,16 @@ export function getShapeAs3D(shape: number[]): [number, number, number] {
 export function getTextureShapeFromLogicalShape(
     logShape: number[], isPacked = false): [number, number] {
   let maxTexSize = env().getNumber('WEBGL_MAX_TEXTURE_SIZE');
+  let maxSizeForNarrorTex =
```

Comment:
```suggestion
  let maxSizeForNarrowTex =
```
(same everywhere else)

Reply:
Thanks! Done.